postgresql/postgresql_installer.sh

```diff
@@ -323,9 +323,9 @@ do_start()
 
 do_stop()
 {
-    PIDS="\`ps aux|grep \$DAEMON | grep \$CONFIGFILE | awk '{print \$2}'\`"
-    if [ "\$PIDS" != "" ]; then
-        kill \$PIDS
+    PGID="\`ps ax o pgid,args |grep \$DAEMON | grep \$CONFIGFILE | awk '{print \$1}'\`"
```

Comment:
o appears to be an illegal argument. should it be -o, instead?
`ps ax -o pgid,args`